MoEngagePluginBase/MoEngagePluginBridge.swift

```diff
@@ -23,7 +24,7 @@ import MoEngageInApps
            let messageHandler = MoEngagePluginMessageDelegate.fetchMessageQueueHandler(identifier: identifier) {
             messageHandler.flushAllMessages()
         }
-        
+        shouldTrackUserAttributeBooleanAsNumber = MoEngagePluginUtils.fetchShouldTrackUserAttrBooleanAsNumberFromPayload(attribute: accountInfo)
```

Comment:
As I understand this flag should be instance specific, shouldn't the appID be checked here?

Reply:
Addressed. Yes there should be a check for appID as well.